### DISCUSSION TITLE
Sísifo

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#322] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/sisifo/

**Comment by brunoaugusto** [comment]
Sísifo, o maior fã dos Rolling Stones de todos os tempos :rofl:

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/sisifo/#comment_1128957740>):

> Sísifo, o maior fã dos Rolling Stones de todos os tempos 🤣

Não entendi

**Comment by najarin-the-loremaster** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/sisifo/#comment_1129003613>):

> Não entendi

Alguns dos contos mitológicos são realmente engraçados, mesmo que bem longos e difíceis de acompanhar:



Zeus ordenou que Thanatos — ou Hades, dependendo a versão — acorrentasse a alma de Sísifo no Tártaro. Sísifo astuto pediu para ver como as correntes funcionavam e, assim, acabou acorrentando seu executor, o que fez com que ninguém mais na Terra morresse. Isso irritou Ares porque suas batalhas perderam a diversão, já que ninguém morria.



Antes de morrer, Sísifo pediu que sua esposa como "prova de amor" lhe concedesse um último desejo, o de ter seu corpo nu lançado em praça pública, o que seu corpo fosse parar às margens do Rio Estígio que faz divisa com o Submundo. Quando sua alma atravessou, ele persuadiu a "esposa" de Hades, Persephone, que permitiu que ele voltasse ao mundo superior. De volta, ele repreendeu sua esposa por não enterrar seu corpo e quando sua alma se recusou a voltar pro Submundo, o mensageiro dos deuses, Hermes, o arrastou de volta a força.



Finalmente preso, Sísifo foi condenado a passar toda a eternidade empurrando uma enorme rocha até o topo de uma colina. Hades mostrando um pouco de sua própria esperteza, encantou a rocha para que sempre que chegasse ao topo rolasse pra longe de Sísifo.



Fonte: [Wikipédia](<https://en.wikipedia.org/wiki/Sisyphus>) (em inglês)

**Comment by one-eyed-willy** [comment]
> [Najarin, the Loremaster](</perfil/?user=najarin-the-loremaster>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/sisifo/#comment_1129101858>):

> Alguns dos contos mitológicos são realmente engraçados, mesmo que bem longos e difíceis de acompanhar:Zeus ordenou que Thanatos — ou Hades, dependendo a versão — acorrentasse a alma de Sísifo no Tártaro. Sísifo astuto pediu para ver como as correntes funcionavam e, assim, acabou acorrentando seu executor, o que fez com que ninguém mais na Terra morresse. Isso irritou Ares porque suas batalhas perderam a diversão, já que ninguém morria.Antes de morrer, Sísifo pediu que sua esposa como "prova de amor" lhe concedesse um último desejo, o de ter seu corpo nu lançado em praça pública, o que seu corpo fosse parar às margens do Rio Estígio que faz divisa com o Submundo. Quando sua alma atravessou, ele persuadiu a "esposa" de Hades, Persephone, que permitiu que ele voltasse ao mundo superior. De volta, ele repreendeu sua esposa por não enterrar seu corpo e quando sua alma se recusou a voltar pro Submundo, o mensageiro dos deuses, Hermes, o arrastou de volta a força.Finalmente preso, Sísifo foi condenado a passar toda a eternidade empurrando uma enorme rocha até o topo de uma colina. Hades mostrando um pouco de sua própria esperteza, encantou a rocha para que sempre que chegasse ao topo rolasse pra longe de Sísifo.Fonte: [Wikipédia](<https://en.wikipedia.org/wiki/Sisyphus>) (em inglês)

![null](<https://i.postimg.cc/2jMbWGYB/nemly-e-nemlerey.png>)



Mentira, eu li sim e foi realmente da hora. Deu até vontade de assistir aquela série da Netflixo agora :grin:

**Comment by brunoaugusto** [comment]
> [Najarin, the Loremaster](</perfil/?user=najarin-the-loremaster>) disse [terça-feira, 17 de maio de 2022](<https://www.vish.blog.br/imagens/tirinhas/sisifo/#comment_1129101858>):

> Alguns dos contos mitológicos são realmente engraçados, mesmo que bem longos e difíceis de acompanhar:Zeus ordenou que Thanatos — ou Hades, dependendo a versão — acorrentasse a alma de Sísifo no Tártaro. Sísifo astuto pediu para ver como as correntes funcionavam e, assim, acabou acorrentando seu executor, o que fez com que ninguém mais na Terra morresse. Isso irritou Ares porque suas batalhas perderam a diversão, já que ninguém morria.Antes de morrer, Sísifo pediu que sua esposa como "prova de amor" lhe concedesse um último desejo, o de ter seu corpo nu lançado em praça pública, o que seu corpo fosse parar às margens do Rio Estígio que faz divisa com o Submundo. Quando sua alma atravessou, ele persuadiu a "esposa" de Hades, Persephone, que permitiu que ele voltasse ao mundo superior. De volta, ele repreendeu sua esposa por não enterrar seu corpo e quando sua alma se recusou a voltar pro Submundo, o mensageiro dos deuses, Hermes, o arrastou de volta a força.Finalmente preso, Sísifo foi condenado a passar toda a eternidade empurrando uma enorme rocha até o topo de uma colina. Hades mostrando um pouco de sua própria esperteza, encantou a rocha para que sempre que chegasse ao topo rolasse pra longe de Sísifo.Fonte: [Wikipédia](<https://en.wikipedia.org/wiki/Sisyphus>) (em inglês)

Por isso eu disse *rolling... stones* 😁



[![null](<https://camo.githubusercontent.com/56ae1a2f6781b7342395aa4ef30a7ea250e0400b566f5532a43aca50b4255315/68747470733a2f2f692e706f7374696d672e63632f4c367250647038582f73697369666f2e706e67>)](<https://camo.githubusercontent.com/56ae1a2f6781b7342395aa4ef30a7ea250e0400b566f5532a43aca50b4255315/68747470733a2f2f692e706f7374696d672e63632f4c367250647038582f73697369666f2e706e67>)
